lib/controllers/books.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require('express');
+const Book = require('../models/Book');
+
+module.exports = Router()
+  .post('/:id', async (req, res, next) => {
+    try {
+      const book = await Book.insert(req.body);
+      res.json(book);
+    } catch (e) {
+      next(e);
+    }
+  })
```

Comment:
Julie had accidentally placed post /books/:id, we will actually want this to be /books in the future to follow rest conventions.